README.md

```diff
@@ -1,146 +1,164 @@
+      
+     The best practise to share your platform configurations is by **sharing the encrypted version of `production.yaml`**.  
 
-## Authentication for monitoring
-If you want to have authentication you need to generate it somehow and there are couple of ways to do that. You can use these websites:
+### Configure
 
-http://www.htaccesstools.com/htpasswd-generator/
+#### Project Structure
+- [/bin](bin): Contains initialization scripts
+- [/charts](charts): Contains Helm Charts for RADAR-base components
+- [/cp-helm-charts](cp-helm-charts): Path where Helm charts from Confluent platform will be downloaded using git
+- [/etc](etc): Contains Helm charts for third party applications used in RADAR-base
+- [/helmfile.d](helmfile.d): Contains Helmfiles for modular deployment of the platform
+- [environment.yaml](environments.yaml): Defines current environment files in order to be used by helmfile. Read more about `bases` [here](https://github.com/roboll/helmfile).
+- [production.yaml](production.yaml): Production helmfile template to configure and install RADAR-base component. Inspect the file to enable, disable and configure components required for your usecase. The default helmfile enables all core components that are needed to run RADAR-base platform with pRMT and aRMT apps. If you're not sure which components you want to enable you can refer to wiki for [an overview and breakdown on RADAR-Base components and their roles](https://radar-base.atlassian.net/wiki/spaces/RAD/pages/2673967112/Component+overview+and+breakdown). 
+- [production.yaml.gotmpl](production.yaml.gotmpl): Change setup parameters that require Go templating, such as reading input files
 
-https://www.web2generators.com/apache-tools/htpasswd-generator
+1. Configure the [enviornments.yaml](environments.yaml) to use the files that you have created by copying the template files.
+    ```shell script
+    vim environments.yaml # use the files you just created
+    ```
+2. Configure the [production.yaml](production.yaml). In this file you are required to fill in secrets and passwords used by RADAR-base applications. It is strongly recommended to use random password generator to fill these secrets. **You must keep this file secure and confidential once you have started installing the platform.**
+    
+    To create an encrypted password string and put it inside `kube_prometheus_stack.nginx_auth` variable. Please make sure you are using MD5 encryption. It appears `bcrypt` encryption isn't supported in current ingress-nginx.
+    
+    Optionally, you can also enable or disable other components that are configured otherwise by default.
+  
+    ```shell script
+    vim production.yaml  # Change setup parameters and configurations
+    ```
+3. In [production.yaml.gotmpl](production.yaml.gotmpl) file, change setup parameters that require Go templating, such as reading input files
```

Comment:
has `production.yaml.gotmp` been added to the repo parent folder

Reply:
It should be created in the `Prepare` stage, by copying the `base.yaml.gotmp`.